kubetest2-aks/cluster-templates/autoscaling-multipool.json

```diff
@@ -37,6 +37,10 @@
         "minCount": 1
       }
     ],
+    "servicePrincipalProfile": {
```

Comment:
No sp profile is by design because we want to use msi later. If you need it, please use your own template.

Reply:
I have removed this.